OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -169,9 +169,14 @@ public void SendAirstrike(Actor self, WPos target, bool randomize = true, int at
 					beacon = new Beacon(
 						self.Owner,
 						target - new WVec(0, 0, altitude),
-						Info.BeaconPalettePrefix,
+						Info.BeaconPaletteIsPlayerPalette,
+						Info.BeaconPalette,
+						Info.BeaconImage,
 						Info.BeaconPoster,
 						Info.BeaconPosterPalette,
+						Info.CircleSequence,
+						Info.ArrowSequence,
```

Comment:
You've got the order of the arguments flipped around here: `Info.ArrowSequence` comes before `Info.CircleSequence`.